src/elements/OcSpinner.vue

```diff
@@ -1,21 +1,32 @@
 <template>
-  <div :uk-spinner="'ratio:' + ratio" :aria-label="ariaLabel"></div>
+  <div :uk-spinner="'ratio:' + ratio" :aria-label="ariaLabel" tabindex="-1" role="img"></div>
 </template>
 
 <script>
 /**
  * Remote actions can take an undefined portion of time. The spinner gives feedback to the users about an actions being processed.
+ *
+ * # Accessibility
```

Comment:
I'd prefer if we wouldn't start with h1. Since we will get later on some more documentation from other categories I think it would be better to have all of them with h2. Also shouldn't be h1 the name of the component in the header? Looking now that it is h2 (But this is a question for different issue/PR. That of course if it really is an issue.)...

```suggestion
 * ## Accessibility
```

Reply:
Fixed